dll/win32/newdev/wizard.c

```diff
@@ -651,15 +683,21 @@ CHSourceDlgProc(
 
                 case IDC_BROWSE:
                 {
-                    BROWSEINFO bi = { 0 };
+                    BROWSEINFOW bi = { 0 };
                     LPITEMIDLIST pidl;
                     WCHAR Title[MAX_PATH];
+                    WCHAR CustomSearchPath[MAX_PATH];
+                    INT idx = SendDlgItemMessageW(hwndDlg, IDC_COMBO_PATH, CB_GETCURSEL, 0, 0);
```

Comment:
On LiveCD the `idx` is always equal to `-1` aka `CB_ERR`, and then CB_GETLBTEXT receives garbage.